class.jetpack-options.php

```diff
@@ -321,4 +320,59 @@ private static function delete_grouped_option( $group, $names ) {
 		return true;
 	}
 
+	static function delete_raw_option( $name ) {
+		global $wpdb;
+		$wpdb->query( $wpdb->prepare( "DELETE FROM $wpdb->options WHERE option_name = %s", $name ) );
+	}
+
+	// Raw option methods allow Jetpack to get / update / delete options via direct DB queries, including options
+	// that are not created by the Jetpack plugin. This is helpful when we need to bypass cache and filters.
+
+	static function update_raw_option( $name, $value, $autoload = false ) {
+
+		$autoload_value = $autoload ? 'yes' : 'no';
+
+		// we write our own option updating code to bypass filters/caching/etc on set_option/get_option
+		global $wpdb;
```

Comment:
I normally prefer `global` to be the first line in a function so it doesn't get missed, but I'm not sure if its a codified standard.  Probably fine, but worth mentioning.